### ISSUE TITLE
handle color support for scoreboards

### ISSUE THREAD

**Issue by ajefts** [topcoder-platform/leaderboard-ui#7]
Add a way to set the members handle color.  I'm ok if you just want to make it a json field for now like we are storing the other user properties.  This would color their handle on the leaderboard.

**Comment by ajefts** [comment]
fyi @callmekatootie

**Comment by callmekatootie** [comment]
@ajefts The `primaryColor` attribute controls the color of the page's theme.

The shade on the header, the background images to use for the participants and the member handles use this color - this is as per the design shared.

Are you asking for a separate color specific for member handles alone?

**Comment by ajefts** [comment]
Yes, I’d like to be able to color the member handles based on their rating. This isn’t a must have for next week, but we should support rating colors in the future.
